### ISSUE TITLE
language should also support two part

### ISSUE THREAD

**Issue by rpenny125** [zowe/zlux-app-manager#27]
e.g., if the user chooses US english (either explicitly, or implicilty through their browser), their "language" would be en-US. The translation system should be able to load messages.en-US.xlf if it's there, and fall back to messages.en.xlf if the "more specific" translation file is not available.

**Comment by 1000TurquoisePogs** [comment]
Isn't this now true? https://github.com/zowe/zlux-app-manager/commit/f0c44cfce04b8d8148311ece7ff57a7f4cdb1804

**Comment by 1000TurquoisePogs** [comment]
This half-works. Some items do not get correctly translated as a result of missing the first check for an internationalized file. We'll need to take another shot at this shortly.
![image](https://user-images.githubusercontent.com/30730276/47365630-f56a4680-d6a9-11e8-91fa-d65ba41ed323.png)

**Comment by 1000TurquoisePogs** [comment]
Yes if you understand the issue please open a ticket for the angular-l10n group so that we get an answer. For us we can just note it here.
